src/voigt.jl

```diff
@@ -131,9 +131,18 @@ Base.@propagate_inbounds function tovoigt!(v::AbstractMatrix{T}, A::SymmetricTen
     return v
 end
 
+@inline function tomandel(A::Tensor{o, dim, T}; order=DEFAULT_VOIGT_ORDER[dim]) where {o, dim, T}
+    return @inbounds tovoigt(A; order=order)
+end
 @inline function tomandel(A::SymmetricTensor{o, dim, T}; order=DEFAULT_VOIGT_ORDER[dim]) where {o, dim, T}
     return @inbounds tovoigt(A; offdiagscale=T(√2), order=order)
 end
+Base.@propagate_inbounds function tomandel!(v::AbstractVector{T}, A::Tensor{2, dim}; offset::Int=0, order=DEFAULT_VOIGT_ORDER[dim]) where {dim, T}
+    tovoigt!(v, A; offset=offset, order=order)
+end
+Base.@propagate_inbounds function tomandel!(v::AbstractMatrix{T}, A::Tensor{4, dim}; offset_i::Int=0, offset_j::Int=0, order=DEFAULT_VOIGT_ORDER[dim]) where {dim, T}
+    tovoigt!(v, A; offset_i=offset_i, offset_j=offset_j, order=order)
+end
```

Comment:
Would line 140 in the latter suggested change suffice for all (non-symmetric) Tensors? In that case, does it work with having `v` as `AbstractVector` or should it be `AbstractArray`? Or should all three functions be written and this only called for the `Tensor{1}` case? In that case, for my own understanding, why would we not have `Tensor{1}` specifically?

Reply:
I guess the "correct" way would be to define
```
tomandel!(v::AbstractVector, ::Tensor{1})
tomandel!(v::AbstractVector, ::Tensor{2})
tomandel!(v::AbstractMatrix, ::Tensor{4})
```
but since this just calls `tovoigt!`, which will then fail for the combination `AbstractVector` `Tensor{4}`  it should be fine to simply define
```
tomandel!(v::AbstractVecOrMat, ::Tensor)
```